src/routes/Dashboard/CardList/cardList.module.scss

```
@@ -0,0 +1,52 @@
+@use 'styles/constants/_colors.scss';
+
+.cardListWrapper {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(3, 1fr);
+  width: 100%;
+  height: 100%;
+  height: 500px;
+}
+
+.cardWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 370px;
+  height: 80px;
+  padding: 18px 40px;
+  margin: 0 90px;
+  border: 1px solid colors.$GRAYA;
+  border-radius: 10px;
+
+  .cardTitle {
+    margin-bottom: 10px;
+    font-size: 15px;
+    line-height: 14px;
+    color: colors.$GRAYA;
```

```suggestion
  border: 1px solid colors.$GRAY300;
  border-radius: 10px;

  .cardTitle {
    margin-bottom: 10px;
    font-size: 15px;
    line-height: 14px;
    color: colors.$GRAY300;
```

저희 아까 grayscale 색상값들 다 날려가지고 이 부분 머지되면 팔레트 색상으로 변경해야합니당